plugin/pulpcore/plugin/changeset/main.py

```diff
@@ -61,10 +60,12 @@ class ChangeSet:
         >>>
     """
 
-    def __init__(self, importer, additions=(), removals=()):
+    def __init__(self, importer, version, additions=(), removals=()):
```

Comment:
Can this become s/version/repo_version/ ? I think that would be clearer and also the same name as the attribute.